docs/current_docs/features/caching.mdx

```diff
@@ -9,8 +9,8 @@ One of Dagger's most powerful features is its ability to cache data across workf
 
 Dagger caches two types of data:
 
-1. Layers: This refers to build instructions and the results of some API calls. This cache is implemented by Buildkit.
-2. Volumes: This refers to the contents of a Dagger filesystem volume and is persisted across Dagger Engine sessions. It is implemented by Dagger (distinct from Buildkit).
+1. **Layers**: This refers to build instructions and the results of some API calls. This cache is implemented by Buildkit
```

Comment:
IMHO we shouldn't be using bold/italic formatting without first setting a standard rule for usage. This is because different authors have different views on what should be emphasized, and the end result is UX inconsistency over time, as formatting styles get applied differently in different pages.  I'd be happy to work on defining these rules in our [style guide](https://github.com/dagger/dagger/blob/main/docs/STYLE_GUIDE.md) as a first step.